source/local/local.go

```diff
@@ -66,7 +77,9 @@ type Config struct {
 
 // parsingOpts contains options used for directives parsing
 type parsingOpts struct {
-	ExpiryTime time.Time
+	ExpiryTime           time.Time
+	IsIncludedInLabels   bool
+	IsIncludedInFeatures bool
```

Comment:
I'd suggest to inverse the logic of the opts. So that the default value (`false`) produces the default behavior (i.e. no directive specified)

```suggestion
	SkipLabel   bool
	SkipFeature bool
```

(field names don't need to be `SkipLabel` and `SkipFeature`, I just picked them as short alternatives...)